### PR TITLE
Update show PDF password components

### PR DESCRIPTION
**Pull request by Mottie**
## Description

The `ShowPdfPassword` component needs to be updated to use web components. It is currently using React components which is causing it to render a "required" label with extra parentheses and an asterisk
required Input label with the required text wrapped in extra parentheses and an asterisk

This PR switches to use `va-text-input` and `va-button`, which both fixes the required label problem and makes the "Add password" button blue (since the green style has been deprecated)

## Original issue(s)

[#46083](https://github.com/department-of-veterans-affairs/va.gov-team/issues/46083)

## Testing done

Updated unit tests

## Screenshots

After

PDF password input with proper required label (no duplicate symbols) and a blue Add password button below it

## Acceptance criteria
- [x] Switch to use web components
- [x] Updated tests
- [x] All tests passing

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by micahchiang — APPROVED** [review]

File: src/platform/forms-system/src/js/utilities/file/ShowPdfPassword.jsx [hunk 1]
```diff
@@ -1,56 +1,60 @@
 import React, { useState } from 'react';
-import TextInput from '@department-of-veterans-affairs/component-library/TextInput';
+import PropTypes from 'prop-types';
+import {
+  VaTextInput,
+  VaButton,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
```
Comment (bkjohnson) [inline src/platform/forms-system/src/js/utilities/file/ShowPdfPassword.jsx:6]: I'm pretty sure that we don't need a React binding for `VaTextInput` - native event handlers like `onInput` work just fine without the binding:

https://github.com/department-of-veterans-affairs/vets-website/blob/eac1ae10b3c9670b765ce2aea6e5546cc694c62c/src/applications/financial-status-report/components/PreSubmitSignature.jsx#L142-L147

`onBlur` and the `onClick` for the button should work the same way.

**Review by bkjohnson — COMMENTED** [review]
Using a React binding for a web component comes at a small performance cost, so we might not want to use them unless absolutely necessary.

File: src/platform/forms-system/src/js/utilities/file/ShowPdfPassword.jsx (hunk 1 above)
Comment (Mottie) [reply src/platform/forms-system/src/js/utilities/file/ShowPdfPassword.jsx:6]: Oh, good to know, thanks @bkjohnson!

**Comment by bkjohnson** [comment]
This should fix the tests since my suggestion broke them :upside_down_face: 

```diff
diff --git a/src/platform/forms-system/test/js/utilities/file.unit.spec.js b/src/platform/forms-system/test/js/utilities/file.unit.spec.js
index bd91ef40b3..5c041111d8 100644
--- a/src/platform/forms-system/test/js/utilities/file.unit.spec.js
+++ b/src/platform/forms-system/test/js/utilities/file.unit.spec.js
@@ -282,8 +282,9 @@ describe('ShowPdfPassword', () => {
     const { container } = render(<ShowPdfPassword {...props} />);
     fireEvent.click($('va-button', container), buttonClick);
 
-    expect($('va-text-input', container).error).to.contain(
-      'provide a password',
+    expect($('va-text-input', container)).to.have.attr(
+      'error',
+      'Please provide a password to decrypt this file',
     );
   });
   it('should call onSubmitPassword', () => {
@@ -293,7 +294,7 @@ describe('ShowPdfPassword', () => {
 
     fireEvent.click($('va-button', container), buttonClick);
 
-    expect($('va-text-input', container).error).to.be.null;
+    expect($('va-text-input', container)).not.to.have.attr('error');
     expect(props.onSubmitPassword.calledOnce).to.be.true;
     expect(props.onSubmitPassword.args[0]).to.deep.equal([
       { name: 'foo' },
```

**Pull request merged**